applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -24,49 +24,81 @@ LOG_MODULE_REGISTER(bis_headset, CONFIG_BLE_LOG_LEVEL);
 BUILD_ASSERT(CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT <= 2,
 	     "A maximum of two broadcast streams are currently supported");
 
+struct audio_codec_info {
+	uint8_t id;
+	uint16_t cid;
+	uint16_t vid;
+	int frequency;
```

Comment:
Does these have to be int? I'm guessing most of these can be uint8_t and then bitrate and frequency probably needs to be uint32_t